From 3b35445238aae4a03f29a49809bf365d66895d6d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 23 Mar 2020 20:05:47 +0200
Subject: [PATCH] [SW-10764] preboot: move to former spl offset

The Preboot binary was placed additionally to SPL offset at the exactly
same time the SPL was removed. Thus currently (for smoother switch) we
have Preboot binary at 2 offsets:
Goya and Gaudi: 0x1A0000 and 0xA0000
Goya2         : 0x1C0000 and 0xC0000

The second image will live for some time (for smoother switch) and
eventually will be removed as a part of reduction of the SPI image size.
Switch to use Preboot binary at the lower offset - former SPL offset.

Change-Id: Iebed054a9d6ddf3a9f543993e0c17134ec1c4080
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index cbeb9eb..f49d70f 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -9,8 +9,8 @@
 
 #define HL_SPI_NEXT_IMAGE_OFF	0x60000
 #define HL_SPI_MODIFI_IMAGE_OFF	0x60004
+#define HL_SPI_PREBOOT_OFF	0xA0000
 #define HL_SPI_UBOOT_OFF	0xB0000
-#define HL_SPI_PREBOOT_OFF	0x1A0000
 
 #define PPBOOT_TEXT_BASE		0x7FF0000000
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
diff --git a/goya/common.h b/goya/common.h
index e985bb0..70b6cc8 100644
--- a/goya/common.h
+++ b/goya/common.h
@@ -9,8 +9,8 @@
 
 #define HL_SPI_NEXT_IMAGE_OFF	0x60000
 #define HL_SPI_MODIFI_IMAGE_OFF	0x60004
+#define HL_SPI_PREBOOT_OFF	0xA0000
 #define HL_SPI_UBOOT_OFF	0xB0000
-#define HL_SPI_PREBOOT_OFF	0x1A0000
 
 #define PPBOOT_TEXT_BASE	0x7FF0000000
 #define PREBOOT_TEXT_BASE	(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
diff --git a/goya2/common.h b/goya2/common.h
index fc28d4b..9479f6c 100644
--- a/goya2/common.h
+++ b/goya2/common.h
@@ -9,8 +9,8 @@
 
 #define HL_SPI_NEXT_IMAGE_OFF	0x80000
 #define HL_SPI_MODIFI_IMAGE_OFF	0x80004
+#define HL_SPI_PREBOOT_OFF	0xC0000
 #define HL_SPI_UBOOT_OFF	0xD0000
-#define HL_SPI_PREBOOT_OFF	0x1C0000
 
 #define PPBOOT_TEXT_BASE		0x7FF0000000
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
diff --git a/goya2_arc/common.h b/goya2_arc/common.h
index 9ce5239..a13067e 100644
--- a/goya2_arc/common.h
+++ b/goya2_arc/common.h
@@ -9,8 +9,8 @@
 
 #define HL_SPI_NEXT_IMAGE_OFF	0x80000
 #define HL_SPI_MODIFI_IMAGE_OFF	0x80004
+#define HL_SPI_PREBOOT_OFF	0xC0000
 #define HL_SPI_UBOOT_OFF	0xD0000
-#define HL_SPI_PREBOOT_OFF	0x1C0000
 
 #define PPBOOT_TEXT_BASE	0x1B05000C
 #define PREBOOT_TEXT_BASE	0x8
